### PR TITLE
Fix resizing too small causing an index out of range crash

### PR DESCRIPTION
**Pull request by coolguy1842**
I haven't verified the change too much, but it seems to work well.

### CODE REVIEW

**Review by Trial97 — APPROVED** [review]

File: launcher/ui/instanceview/VisualGroup.cpp [hunk 1]
```diff
@@ -66,6 +66,9 @@ void VisualGroup::update()
             rows[currentRow].height = maxRowHeight;
             rows[currentRow].top = offsetFromTop;
             currentRow++;
+            if(currentRow >= rows.size()) {
```
Comment (Trial97) [inline launcher/ui/instanceview/VisualGroup.cpp]:
```suggestion
            if (currentRow >= rows.size()) {
```

**Comment by Trial97** [comment]
And you need to sign your commits  see https://github.com/PrismLauncher/PrismLauncher/pull/2474/checks?check_run_id=25876625758

File: launcher/ui/instanceview/VisualGroup.cpp (hunk 1 above)
Comment (coolguy1842) [reply launcher/ui/instanceview/VisualGroup.cpp:69]: Hey, I thought I did but ended up doing it wrongly, Will try to fix now.

**Comment by coolguy1842** [comment]
Hey, I thought I did but ended up doing it wrongly, Will try to fix now.

**Review by TayouVR — APPROVED** [review]

**Pull request merged**